app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -21,7 +21,8 @@ <h1>Reader Study {{ object.title }}</h1>
 
     {% if not object.is_valid %}
         <div class="alert alert-danger">
-        This reader study is not ready to be used. Please update the configuration.
+        This reader study is not ready to be used. Please update the configuration. All of the images should be included in the hanging list
+        exactly once.
```

Comment:
There are actually more requirements than this, I would remove this message as we need to determine the exact reason why the validation has failed.